MdePkg/Include/Pi/PiHob.h

```diff
@@ -232,7 +232,8 @@ typedef UINT32 EFI_RESOURCE_TYPE;
 #define EFI_RESOURCE_MEMORY_MAPPED_IO_PORT  0x00000004
 #define EFI_RESOURCE_MEMORY_RESERVED        0x00000005
 #define EFI_RESOURCE_IO_RESERVED            0x00000006
-#define EFI_RESOURCE_MAX_MEMORY_TYPE        0x00000007

```

Comment:
Since EFI_RESOURCE_MEMORY_UNACCEPTED has not been officially published in PI spec, it cannot be declared in MdePkg. (This is told by the MdePkg maintainer -- Liming Gao <gaoliming@byosoft.com.cn>.) Instead it can be defined in MdeModulePkg.

Reply:
I am afraid it will not be published in short-term. (maybe next year.)
But we still can define it in MdeModulePkg.